Pulled in from devel/abseil-cpp to fix the ppc64 build

Index: deps/v8/third_party/abseil-cpp/absl/base/internal/unscaledcycleclock_config.h
--- deps/v8/third_party/abseil-cpp/absl/base/internal/unscaledcycleclock_config.h.orig
+++ deps/v8/third_party/abseil-cpp/absl/base/internal/unscaledcycleclock_config.h
@@ -21,7 +21,7 @@
 
 // The following platforms have an implementation of a hardware counter.
 #if defined(__i386__) || defined(__x86_64__) || defined(__aarch64__) || \
-    defined(__powerpc__) || defined(__ppc__) || defined(__riscv) ||     \
+    /* defined(__powerpc__) || defined(__ppc__) || */ defined(__riscv) ||     \
     defined(_M_IX86) || (defined(_M_X64) && !defined(_M_ARM64EC))
 #define ABSL_HAVE_UNSCALED_CYCLECLOCK_IMPLEMENTATION 1
 #else
